frontend/src/components/layout/InputValidationMessage.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="text-red-fire font-semibold tracking-wide pt-1 px-5">
+    <slot>{{ message }}</slot>
```

Comment:
The slot was mainly added to support the fixed height display of the error message together with its empty version here: https://github.com/beamer-bridge/beamer/pull/918/files/1ace43a3430fe241172f5cbedf11044314f7a3ce#diff-8c02af51cfb483f693670bd4e1b2c8a9963ba212bc1777c0df643e0cd9432dbeR40-R44

Wrote it this way in order to prevent the form cards from moving vertically once the error message gets displayed due to their fixed height limits.
We can also take the solution for this outside of the `InputValidationMessage.vue` and inside `RequestTargetInputs.vue`  by applying the same padding classes to the empty `div` substitution (when error message is hidden) but that way we would need to sync the padding classes each time they are changed inside the `InputValidationMessage.vue` component.

I also thought it might not hurt to give a more configurable approach to the validation message by allowing control over it both via a message prop and a slot for custom design if needed.